tests/utils.py

```diff
@@ -60,36 +61,40 @@ class RemoteOpenAIServer:
 
     def __init__(self,
                  model: str,
-                 cli_args: List[str],
+                 serve_args: List[str],
```

Comment:
why rename the arg name btw?

Reply:
I meant to associate them with `vllm serve` specifically, not just any CLI. Perhaps it could be clearer.